Libplanet.Unity/MonoSingleton.cs

```diff
@@ -0,0 +1,107 @@
+#nullable disable
+using UnityEngine;
+
+namespace Libplanet.Unity
+{
+    /// <summary>
+    /// MonoSingleton.
+    /// </summary>
+    /// <typeparam name="T">The type.</typeparam>
```

Comment:
```suggestion
    /// <summary>
    /// A singleton implementation with <see cref="MonoBehaviour"/> as its base class.
    /// </summary>
    /// <typeparam name="T">The class to wrap with this singleton pattern.</typeparam>
```

I'd say single word summaries are going bit too far. 😐

Reply:
Oops, I'm sorry. I forgot to write the document
Thanks you 👍